weave-js/src/components/PagePanelComponents/Home/Browse3/typeViews/PIL.Image.Image/PILImageImage.tsx

```diff
@@ -21,11 +21,25 @@ export const PILImageImage: React.FC<{
   data: PILImageImageTypePayload;
 }> = props => {
   const {useFileContent} = useWFHooks();
+
+  const imageTypes = {
+    'image.jpg': 'jpg',
+    'image.png': 'png',
+  } as const;
+
+  const imageKey = Object.keys(props.data.files).find(
+    key => key in imageTypes
+  ) as keyof PILImageImageTypePayload['files'];
+  if (!imageKey) {
+    return <span>Img not found!</span>;
```

Comment:
I would use `weave-js/src/components/PagePanelComponents/Home/Browse2/NotApplicable.tsx` this component